### PR TITLE
Fix UTF-8 encoding for copyright in About dialog

### PR DESCRIPTION
**Pull request by DarkMessiah**
Fix About Godot after https://github.com/godotengine/godot/pull/70885

| Before | After |
| --- | --- |
| image| image|

### CODE REVIEW

File: editor/editor_about.cpp [hunk 1]
```diff
@@ -154,7 +154,7 @@ EditorAbout::EditorAbout() {
 	Label *about_text = memnew(Label);
 	about_text->set_v_size_flags(Control::SIZE_SHRINK_CENTER);
 	about_text->set_text(String::utf8("\xc2\xa9 2014-present ") + TTR("Godot Engine contributors") +
```
Comment (akien-mga) [inline editor/editor_about.cpp:156]: Could you also add a period at the end of this one? Seems I forgot it.

**Comment by akien-mga** [comment]
Could you make the commit message more explicit? It should specify both where you fix things and *what* you fix. E.g. "Fix UTF-8 encoding for copyright in About dialog".

File: editor/editor_about.cpp (hunk 1 above)
Comment (DarkMessiah) [reply editor/editor_about.cpp:156]: `TTR("Godot Engine contributors") + "."` or `TTR("Godot Engine contributors.")`? :)

Comment (akien-mga) [reply editor/editor_about.cpp:156]: I guess the former as some languages also use a different character for period.

**Comment by DarkMessiah** [comment]
Done!

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**